frontend/public/components/dashboards-page/overview-dashboard/top-consumers-card.tsx

```diff
@@ -78,9 +110,29 @@ const TopConsumersCard_: React.FC<DashboardItemProps> = ({
   const topConsumersResult = prometheusResults.getIn([currentQuery, 'result']);
 
   const stats = getInstantVectorStats(topConsumersResult, topConsumersType.metric, metricTypeSort.humanize);
-
   const data = topConsumersType.mutator ? topConsumersType.mutator(stats) : stats;
 
+  const top5Data = [];
+  const consumersLoaded = _.get(resources.consumers, 'loaded');
```

Comment:
I've opted for 
```suggestion
  const consumersLoaded = _.get(resources, ['consumers', 'loaded']);
```

because of incorrect type detection. When using the dot notation type was `FirehoseResult<K8sResourceKind[]>` which then couldnt be really cast to correct one (boolean/K8sResourceKind[]..). The array notation results in `any` type.

Applied the same for other `_.get` usages